script.js

```diff
@@ -0,0 +1,51 @@
+// завдання 1
+
+function iterativeOddSumTo(number) {
+  let result = 0;
+  for (i = 1; i <= number; i++) {
+    if (i % 2 !== 0) {
```

Comment:
```suggestion
    if (i % 2) {
```
Ідентично, якщо буде 0 то 0 ---> false, і в умову не попадаємо.